### PR TITLE
Feat: manage address and handle payment with paypal

### PR DESCRIPTION
**Pull request by justme2002**

### CODE REVIEW

File: BookEcommerce.Services/Interfaces/IAuthenticationService.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ public interface IAuthenticationService
         public Task<ResponseBase> CustomerRegister(AccountViewModel AccountDTO);
         public Task<ResponseBase> VendorRegister(AccountViewModel AccountDTO);
         public Task<ResponseBase> AdminRegister(AccountViewModel AccountDTO);
-        public Task<ResponseBase> RefreshToken(string Email, TokenViewModel TokenDTO);
+        public Task<TokenResponse> RefreshToken(string Email, TokenViewModel TokenDTO);
```
Comment (Dat070601) [inline BookEcommerce.Services/Interfaces/IAuthenticationService.cs]: you should fix name convention !

File: BookEcommerce/Controllers/AddressController.cs [hunk 2]
```diff
@@ -25,6 +25,8 @@ public async Task<IActionResult> CreateAddress([FromBody] AddressViewModel addre
             var result = await this.addressService.CreateAddress(addressViewModel, authHeader);
             if (!result.IsSuccess)
             {
+                var AuthHeader = Request.Headers["Authorization"].ToString().Split(' ')[1];
```
Comment (Dat070601) [inline BookEcommerce/Controllers/AddressController.cs]: name convention, please !

File: BookEcommerce.Services/Interfaces/IAuthenticationService.cs (hunk 1 above)
Comment (justme2002) [reply BookEcommerce.Services/Interfaces/IAuthenticationService.cs]: done

File: BookEcommerce/Controllers/AddressController.cs (hunk 2 above)
Comment (justme2002) [reply BookEcommerce/Controllers/AddressController.cs]: resolved

**Review by Dat070601 — APPROVED** [review]

**Pull request merged**